include/uapi/sound/sof/tokens.h

```diff
@@ -96,6 +96,7 @@
  */
 #define SOF_TKN_COMP_SINK_PIN_BINDING_WNAME	413
 #define SOF_TKN_COMP_SRC_PIN_BINDING_WNAME	414
+#define SOF_TKN_COMP_ON_BE_PIPELINE		416
```

Comment:
@libinyang this PR only partially addresses your specific problem for triggering the detect pipeline for WoV. It does not address the generic case of triggering multiple pipelines when there are one ore more intermediate pipelines. In such ases, we also need to ensure the order in which the pipelines are triggered. Walking the iwdgets in the DAPM list, will not guarantee the ordering required by the FW

Reply:
> @libinyang this PR only partially addresses your specific problem for triggering the detect pipeline for WoV. It does not address the generic case of triggering multiple pipelines when there are one or more intermediate pipelines. In such cases, we also need to ensure the order in which the pipelines are triggered. Walking the widgets in the DAPM list, will not guarantee the ordering required by the FW

@ranj063 Yes, this patch won't fix the trigger order issue. The trigger order is another issue. I'm planning to create another PR for it.